Hippo.Core/Domain/Order.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Hippo.Core.Domain
+{
+    public class Order
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string Category { get; set; }
+        [Required]
+        [MaxLength(50)]
+        public string Name { get; set; }
+        [MaxLength(250)]
+        public string Description { get; set; }
+        [MaxLength(150)]
+        public string ExternalReference { get; set; }
+        [Required]
+        [Range(0.01, double.MaxValue)]
+        public decimal Price { get; set; }
+        [Required]
+        [Range(0.0001, double.MaxValue)]
+        public decimal Quantity { get; set; }
+        [Required]
+        [Range(1, int.MaxValue)]
+        public int Installments { get; set; }
+
+
+        public decimal Adjustment { get; set; }
+        public string AdjustmentReason { get; set; }
+        public decimal SubTotal { get; set; }
+        public decimal Total { get; set; }
+        public decimal BalanceRemaining { get; set; } // do here, or just get the total of the billings?
```

Comment:
Agree we should do here.  Not against also doing a total for final check, but this is good to keep for display and sanity checks